L1Trigger/L1TMuonEndCap/interface/bdt/Node.h

```diff
@@ -59,7 +59,7 @@ class Node
         Node* filterEventToDaughter(Event* e);
```

Comment:
No changes to this file except spaces ... will probably make the review easier if we just leave it untouched.